Core/ContentBlockerLoader.swift

```diff
@@ -109,11 +121,17 @@ public class ContentBlockerLoader {
             
```

Comment:
I wonder: maybe we also should mark `httpsBloomFilter` as completed if in the above block, as effectively it won't be invoked (and should not be marked as an potential issue).

Reply:
Great catch, thanks!